File: bluetoothProject/app/src/main/java/com/example/bluetoothproject/PumpActivity.java
package com.example.bluetoothproject;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class PumpActivity extends AppCompatActivity {
        @Override
        protected void onCreate(@Nullable Bundle savedInstanceState) {
                super.onCreate(savedInstanceState);
                setContentView(R.layout.activity_pump);

                ImageButton home3 = (ImageButton)findViewById(R.id.home3);
                ImageButton setting3 = (ImageButton) findViewById(R.id.setting3);

                home3.setOnClickListener(new View.OnClickListener() {
                        @Override
                        public void onClick(View view) {
                                finish();
                        }
                });

                setting3.setOnClickListener(new View.OnClickListener() {

                        @Override
                        public void onClick(View view) {
                                /*
                                Intent intent = new Intent(getApplicationContext(), SettingActivity.class);
                                startActivity(intent);

                                 */
                        }


                });

        }
}

File: PlantEngine_TermProject/app/src/main/java/com/example/plantengine_termproject/PumpActivity.kt
package com.example.plantengine_termproject

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.util.Log
import android.view.View
import android.widget.SeekBar
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_pump.*
import kotlinx.android.synthetic.main.activity_water_tank.*
import java.lang.Thread.sleep
import java.util.*
import kotlin.concurrent.schedule

class PumpActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pump)

        ClickLister();
    }
    private fun ClickLister(){
        home3.setOnClickListener {
            finish()
        }
        setting3.setOnClickListener {
            val intent: Intent = Intent(this, SettingActivity::class.java)
            startActivity(intent)
        }
    }
}